components/registry/org.wso2.carbon.registry.indexing/src/main/java/org/wso2/carbon/registry/indexing/solr/SolrClient.java

```diff
@@ -56,14 +56,8 @@
 import java.io.OutputStream;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Hi Thushara,
Thanks for the suggestion. PR [1] sent with changes. 
[1] - https://github.com/wso2/carbon-registry/pull/107